Services/BrokerService/Controllers/BrokerController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using BrokerServices;
+using BrokerServices.Utils;
+using Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BrokerService.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class BrokerController : ControllerBase
+    {
+        [Route("getAllCurrencies")]
+        [HttpGet]
+        public List<IMarketInstrument> GetAllCurrencies([FromBody] BankType bankType)
+        {
+            return BrokerFactory.Create(bankType).GetAllCurrencies();
+        }
+
+        [Route("getCurrency")]
+        [HttpGet]
+        public IMarketInstrument GetCurrency([FromBody] BankType bankType, string idCurrency)
```

Comment:
Why idCurrency is string? Do you mean FIGI? Or what?

Reply:
Because idCurrency is string in TinkoffItegrationLib